fastlane/Fastfile

```diff
@@ -998,6 +1028,7 @@ end
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
+  push_to_git_remote(tags: false)
```

Comment:
I don't remember ever seeing the method to trigger the Buildkite build pushing as well. It makes sense, though, given a build with an out of date branch would be incorrect and if we are not likely to want to trigger a release build for a commit that is not the top of the branch.